### PR TITLE
test: Add TypeScript fixtures

### PR DESCRIPTION
**Pull request by matz3**
JIRA: CPOUI5FOUNDATION-845

### CODE REVIEW

File: test/fixtures/linter/rules/NoDeprecatedApi/NoDeprecatedApiTypeScript.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Configuration.getCompatibilityVersion("sapMDialogWithPadding"); // Method "getCompatibilityVersion" is deprecated
+Configuration["getCompatibilityVersion"]("sapMDialogWithPadding"); // Method "getCompatibilityVersion" is deprecated
+
+Device.browser.webview; // Property "webview" is deprecated
+Device.browser["webview"]; // Property "webview" is deprecated
+
+Configuration.AnimationMode; // Property "AnimationMode" (Enum) is deprecated
+
+MessageType.Error; // TODO detect: Enum "MessageType" is deprecated
```
[the remark this answers is not in the document]
Comment (matz3) [reply test/fixtures/linter/rules/NoDeprecatedApi/NoDeprecatedApiTypeScript.ts:40]: it is a general issue, not related to typescript sources.
The following is also not detected in JS:
```js
const {MessageType} = coreLib;
MessageType.Error;
```

[the remark this answers is not in the document]
Comment (matz3) [reply test/fixtures/linter/rules/NoDeprecatedApi/NoDeprecatedApiTypeScript.ts:40]: See https://github.com/SAP/ui5-linter/pull/224

[the remark this answers is not in the document]
Comment (RandomByte) [reply test/fixtures/linter/rules/NoDeprecatedApi/NoDeprecatedApiTypeScript.ts:40]: Now part of https://github.com/SAP/ui5-linter/pull/261

File: test/fixtures/linter/rules/NoDeprecatedApi/NoDeprecatedApi.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ sap.ui.define([
 	"sap/ui/table/Table", "sap/ui/table/plugins/MultiSelectionPlugin", "sap/ui/core/Configuration", "sap/m/library"
 ], function(Button, DateTimeInput, includes, Device, coreLib, NavigationHandler, Table, MultiSelectionPlugin, Configuration, mobileLib) {
 
-	var dateTimeInput = new DateTimeInput(); // TODO detect: Control is deprecated
```
[the remark this answers is not in the document]
Comment (matz3) [reply test/fixtures/linter/rules/NoDeprecatedApi/NoDeprecatedApi.js:6]: https://github.com/SAP/ui5-linter/pull/274

**Pull request merged**